addons/web/static/src/core/domain_selector/domain_selector.xml

```diff
@@ -23,6 +23,12 @@
                     </t>
                 </t>
             </t>
+            <t t-elif="props.isExpression">
```

Comment:
We have already discussed this. It would better to use composition/inheritance to handle that. An expression is not a domain. Have two props domain and isExpression is really not good imo